src/js/jsx/sections/libraries/LibrariesPanel.jsx

```diff
@@ -100,72 +100,68 @@ define(function (require, exports, module) {
             this.getFlux().actions.libraries.removeCurrentLibrary();
         },
 
-        render: function () {
+        _containerContents: function () {
             var libraryStore = this.getFlux().store("library"),
-                connected = libraryStore.getConnectionStatus(),
-                libraries = this.state.libraries,
-                currentLibrary = libraries.get(this.state.selectedLibrary);
-
-            var containerClasses = classnames({
-                "section-container": true,
-                "section-container__collapsed": !this.props.visible
-            });
-
-            var sectionClasses = classnames({
-                "libraries": true,
-                "section": true,
-                "section__sibling-collapsed": !this.props.visibleSibling
-            });
+                connected = libraryStore.getConnectionStatus();
 
             var containerContents;
-
             if (connected) {
+                var libraries = this.state.libraries,
+                    currentLibrary = libraries.get(this.state.selectedLibrary),
+                    selectedLayers = this.props.document.layers.selected;
+
                 containerContents = this.props.visible && !this.props.disabled && (
-                <div>
-                    <div className="formline">
-                        <LibraryList
-                            ref="libraryList"
-                            libraries={libraries}
-                            selected={currentLibrary}
-                            onLibraryChange={this._handleLibraryChange}
-                        />
-                        <SplitButtonList>
-                            <SplitButtonItem
-                                title={"CREATE LIBRARY"}
-                                className="button-plus"
-                                iconId="plus"
-                                onClick={this._handleLibraryAdd}
-                                />
-                            <SplitButtonItem
-                                title={"DELETE LIBRARY"}
-                                className="button-plus"
-                                iconId="libraries-delete"
-                                onClick={this._handleLibraryRemove}
-                                />
-                        </SplitButtonList>
+                    <div>
+                        <div className="formline">
+                            <LibraryList
+                                libraries={libraries}
+                                selected={currentLibrary}
+                                onLibraryChange={this._handleLibraryChange} />
+                            <SplitButtonList className="libraries__split-button-list">
```

Comment:
I can't find these buttons in the UI anymore. Not important, as they will be replaced by other buttons later.